test/TEST-35-LOGIN/test-session-properties.c

```diff
@@ -0,0 +1,65 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
```

Comment:
I'd like to locate this .c file in src/login.

Reply:
Makes sense, moved.